routes/articleRoutes.js

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+import Articles from '../controllers/article';
```

Comment:
Import this from the index file in the controllers directory as we have been doing before.